README.md

```diff
@@ -7,136 +7,119 @@
 A collection of examples that illustrate how to use Restate to solve common application
 challenges.
 
-* **[Use Cases and Patterns](patterns-use-cases):** Small specific use cases, like webhooks,
-  workflows, asynchronous task queuing.
+* **[Templates](#templates):** Project templates for various languages, build systems and runtimes.
+  Simple 'Hello World!' examples in a proper build setup that you can use if you want to start
+  a brand new project for a service or lambda that will be invoked through Restate.
 
-* **[Basics](basics):** Small examples highlighting the basic building blocks, like
+* **Basics:** Small examples highlighting the basic building blocks, like
   durable execution or virtual objects.
+ 
+* **Use Cases and Patterns:** Small specific use cases, like webhooks,
+  workflows, asynchronous task queuing.
+
+* **Integrations:** Examples of integrating Restate with other popular tools, technologies, and libraries.
 
-* **[End-to-End Applications](end-to-end-applications):** Runnable demo applications that consist
+* **End-to-End Applications:** Runnable demo applications that consist
   of many components, e.g., a food ordering app, or an e-commerce site.
   Typically packaged as a docker compose setup.
 
-* **[Tutorials](tutorials):** A step-by-step guide that builds an application and introduces
+* **Tutorials:** A step-by-step guide that builds an application and introduces
   the Restate concepts on the way.
 
-* **[Templates](templates):** Project templates for various languages and build systems.
-  Simple 'Hello World!' examples in a proper build setup that you can use if you want to start
-  a brand new project for a service or lambda that will be invoked through Restate.
-
+[![TypeScript](https://skillicons.dev/icons?i=ts)](typescript)
```

Comment:
How about we have a readme in the individual "language directories" with the list of examples?